File: DAY_40/DAY_40_01.cpp
// Created by KUNTAL OJHA .
// Today 11/10/2022 (dd/mm/yyyy)
// Problem Link: https://www.codechef.com/problems/LAZYCHF
// Problem Title: Lazy Chef
// Programmng Language: C++



#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while(t--)
	{
	    int x,m,d;
	    cin>>x>>m>>d;
	    
	    if(m*x >= x+d)
	    {
	        cout<<x+d<<endl;
	    }else
	    {
	        cout<<m*x<<endl;
	    }
	}
	return 0;
}

File: DAY_40/DAY_40_01.py
# Created by KUNTAL OJHA .
# Today 11/10/2022 (dd/mm/yyyy)
# Problem Link:
# Problem Title:
# Programmng Language: PYTHON

  
  
def main():
    t = int(input())
    for i in range(t):
        a,b,x,y=map(int,input().split())
        a=a/x
        b=b/y
    
        if a==b:
            print("Both")
        elif a<b:
            print("Chef")
        else:
            print("Chefina")


if __name__=='__main__':
    main()

    
